release-notes/6.0/install.md

```diff
@@ -0,0 +1,102 @@
+# .NET 6 installation instructions
+
+Concise install instructions for Windows are provided in this document, and may be augmented in [release notes](README.md) for a given update. Complete [.NET installation instructions for Windows, macOS, and Linux](https://docs.microsoft.com/dotnet/core/install/) are provided for supported releases in [.NET documentation](https://docs.microsoft.com/dotnet).
```

Comment:
1. Concise instructions are also included for mac and linux.
2. Should the binary installation instructions be included in each of the OS specific instructions?  It feels valuable to include the full set of installation types in one doc per OS.
3. The binary archive instructions don't necessarily feel like the most concise especially to noobs that may be more uncomfortable with cmd line instructions.

Reply:
Good call. Fixed.